twittermap/app/controllers/TwitterMapApplication.scala

```diff
@@ -28,7 +29,11 @@ class TwitterMapApplication @Inject()(val wsClient: WSClient,
   val register = Migration_20170428.migration.up(wsClient, cloudberryRegisterURL)
   Await.result(register, 1 minutes)
 
-  def index = Action {
+  def index = Action { request =>
+    val remoteAddress = request.remoteAddress
+    val userAgent = request.headers.get("user-agent").getOrElse("unknown")
+    Logger.info("user IP address: " + remoteAddress)
+    Logger.info("user agent: " + userAgent)
```

Comment:
wow, it seems easy :-)

can we take this chance to organize the `log`? can we create some customized log so that we can differentiate the system log and the client log, or else? just a thought.